l1-contracts/contracts/governance/L2AdminFactory.sol

```diff
@@ -39,4 +42,13 @@ contract L2AdminFactory {
 
         admin = address(new ChainAdmin{salt: _salt}(restrictions));
     }
+
+    function _validateRestrctions(address[] memory _restrictions) internal view {
```

Comment:
Shall we remove validation from here? The `ChainAdmin` already validates it

Reply:
It will revert earlier though, let's keep it just in case

Reply:
Please add doc comments to the function then + comment above